examples/blake2f.rs

```diff
@@ -0,0 +1,1459 @@
+use chiquito::{
+    frontend::dsl::{
+        cb::{eq, select, table},
+        lb::LookupTable,
+        super_circuit, CircuitContext, StepTypeSetupContext, StepTypeWGHandler,
+    },
+    plonkish::{
+        backend::halo2::{chiquitoSuperCircuit2Halo2, ChiquitoHalo2SuperCircuit},
+        compiler::{
+            cell_manager::{MaxWidthCellManager, SingleRowCellManager},
+            config,
+            step_selector::SimpleStepSelectorBuilder,
+        },
+        ir::sc::SuperCircuit,
+    },
+    poly::ToExpr,
+    sbpir::query::Queriable,
+};
+use halo2_proofs::{
+    dev::MockProver,
+    halo2curves::{bn256::Fr, group::ff::PrimeField},
+};
+use std::hash::Hash;
+
+pub const IV_LENGTH: u64 = 8;
```

Comment:
@10to4 IV_LENGTH could be of type usize for example. If you use it for the length or indexing an array then usize is good. But it does not matter, you can leave it like that.